Dockerfile

```diff
@@ -61,8 +58,12 @@ ENV PATH /edx/app/edxapp/edx-platform/bin:${PATH}
 ENV SETTINGS production
 RUN mkdir -p /edx/etc/
 
+ENV VIRTUAL_ENV=/venv
+RUN python3.8 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
 # Install Python requirements
-RUN pip install pip==20.0.2
+RUN pip install pip==20.2.3 && pip install wheel
```

Comment:
I'd rather keep specific package versions out of the Dockerfile, for reasons mentioned in a couple of recent PRs in other services.  Could we instead create a new requirements file and do something like this?

```docker
COPY requirements/pip.txt requirements/pip.txt
RUN pip install -r requirements/pip.txt
```

That file would specify the current versions of pip, setuptools, and wheel we want to be using (no pip-tools usage needed).  And yes, I'm thinking the file would live at the same level as `constraints.txt` and be used for both regular and codejail virtualenvs.